### ISSUE TITLE
DocumentCardPreview doesn't allow for alt attribute for image

### ISSUE THREAD

**Issue by beatzball** [microsoft/fluentui#21109]
### Environment Information

- **Package version(s)**: latest
- **Browser and OS versions**: any

### Describe the issue:

DocumentCardPreview does not allow for an `alt` attribute to be passed down as a prop and hardcodes it to an empty string `alt=""`

### Please provide a reproduction of the issue in a codepen:

No reproduction of issue necessary as you can see the alt attributes forced to be an empty string defined in code:
- [L88 - DocumentCardPreview.base.tsx](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/DocumentCard/DocumentCardPreview.base.tsx#L88)
- [L94 - DocumentCardPreview.base.tsx](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/DocumentCard/DocumentCardPreview.base.tsx#L94)
- [L127 - DocumentCardPreview.base.tsx](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/DocumentCard/DocumentCardPreview.base.tsx#L127)

#### Actual behavior:

when providing the `alt` HTMLElement prop, nothing happens, but an empty alt attribute is found for the image

#### Expected behavior:

should be able to provide an `alt` prop or something of the like (`imageAlt` maybe?)

**Comment by gouttierre** [comment]
@beatzball - Thanks for filing this issue with us. To set expectations, the developers will review this issue once capacity allows.

@spmonahan - Would you be able to confirm if this is a regression, or if this behavior is an issue? 
cc @smhigley

**Comment by smhigley** [comment]
The original intent was to treat the preview image as purely presentational, hence the lack of alt and the use of `role="presentation"`. We could look into allowing authors to provide an `alt`, defaulting to presentational images if no alt is provided. I'm not really familiar with the range of use cases DocumentCard was originally intended to cover, or if non-presentational images were intended to be used with it.

@beatzball would you be able to provide some use cases where you want to be able to use DocumentCard to show meaningful images?

**Comment by beatzball** [comment]
Thanks for the response, @smhigley. 

Our product was utilizing `DocumentCardPreview` as part of a custom Carousel component, and we were getting reports of a11y issues on the empty alt through FastPass back in December (when we first reported this). 

We missed the correlation of an empty alt being ok with a `role="presentation"`, and FastPass made the update on their end to stop reporting it as an issue. 

I'll close this out since this is not an issue to be fixed.
